src-java/base-topology/base-storm-topology/src/main/java/org/openkilda/wfm/HistoryUpdateCarrier.java

```diff
@@ -25,4 +25,8 @@ public interface HistoryUpdateCarrier {
      * Sends main events to history bolt.
      */
     void sendHistoryUpdate(FlowHistoryHolder historyHolder);
+
+    default String getHistoryStreamName() {
+        return "HUB_TO_HISTORY_TOPOLOGY_SENDER";
+    }
```

Comment:
Yes, I have a thought that it needs to be in some place that is accessible in the bolts and the service at the same time. We need this constant when configuring topologies, so it would be convenient to have some unified approach of adding. If it is a helper method then it can't be enum that we have in the topology package. 
This is a task to think about. But once it is done, we can change this method without changing the rest of the code.

Reply:
I think `getHistoryStreamName()` must be used in `declareOutputFields`